src/routes/product-detail/ProductDetailInsights.tsx

```diff
@@ -61,14 +61,15 @@ export const ProductDetailInsights = (props: Props) => {
         withCloseButton={false}
         size="xl"
       >
-        <Flex mih={700} className="orders-over-time-container">
+        <Flex h="700px" className="orders-over-time-container">
           <InteractiveQuestion
             questionId="jA76uaXQKWC1xp7q2TvE2"
             title={
               <Title fw={400} size="h2" className="product-detail-card-title">
                 Orders over time
               </Title>
             }
+            height="100%"
```

Comment:
Removes the scrollbar on the modal when product insight card is clicked on

<img width="2980" height="1614" alt="Image" src="https://github.com/user-attachments/assets/a2293908-fa72-408b-9680-2017a460b0a4" />